index.html

```diff
@@ -1,19 +1,93 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <div id="root">
+      <!--Navigation section-->
+      <header>
+        <section id="header-container">
+          <img id="logo" src="./img/karma-logo.svg" />
+          <nav>
+            <a href="#"><span style="font-weight: bold">Meet Karma</span></a>
```

Comment:
`<span style="..."> ... ` - same styling could be achieved also with more generic css rule;
```css
nav a:nth-child(1) {
  font-weight: bold;
}
```

see more about it here: [MDN - CSS - nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)